solidity-its/src/main/java/org/sonarsource/solidity/its/SolidityRuling.java

```diff
@@ -91,6 +165,31 @@ public static void deletePreviouslyAnalyzedFiles() {
           LOG.debug(e.getMessage(), e);
         }
       });
+    Files.deleteIfExists(new File(DIFFERENCES).toPath());
+
+  }
+
+  public static void findDifferences() {
+    filesToAnalyze.forEach((projectName, fileList) -> {
+      fileList.stream().forEach(file -> {
+        String projectFileName = file.toString().replaceAll(DIR, "");
+        String actualIssues = SolidityRulingIts.ACTUAL_ISSUES + projectFileName;
+        String recordedIssues = SolidityRulingIts.RECORD_ISSUES + projectFileName;
+        try {
+          System.out.println(recordedIssues + " - " + actualIssues);
+          System.out.println(FileUtils.contentEquals(new File(recordedIssues), new File(actualIssues)));
+          if (!FileUtils.contentEquals(new File(recordedIssues), new File(actualIssues))) {
+            System.out.println("NOT EQUALLLLLLLLLLLLLLLLLLLLLLLLl");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=31&issues=AWRqjT0tMt9UOhrwBY2i&open=AWRqjT0tMt9UOhrwBY2i)

[ISSUE_KEY]: AWRqjT0tMt9UOhrwBY2i